src/index.js

```diff
@@ -280,8 +280,11 @@ class ReactPhoneInput extends React.Component {
     });
   }
 
-  handleInputClick() {
+  handleInputClick(evt) {
     this.setState({showDropDown: false});
+    if (this.props.onClick) {
+      this.props.onClick(evt)
```

Comment:
instead of if condition we can do something like this 

this.props.onClick && this.props.onClick(evt)